orchestrator/gravity_utils/src/types/signatures.rs

```diff
@@ -16,20 +16,12 @@ pub struct GravitySignature {
 }
 
 impl Ord for GravitySignature {
-    // Alex wrote the Go sorting implementation for validator
-    // sets as Greatest to Least, now this isn't the convention
-    // for any standard sorting implementation and Rust doesn't
-    // really like it when you implement sort yourself. It prefers
-    // Ord. So here we implement Ord with the Eth address sorting
-    // reversed, since they are also sorted greatest to least in
-    // the Cosmos module. Then we can call .sort and .reverse and get
-    // the same sorting as the Cosmos module.
+    // Sort by eth address asc
     fn cmp(&self, other: &Self) -> Ordering {
-        if self.power != other.power {
-            self.power.cmp(&other.power)
-        } else {
-            self.eth_address.cmp(&other.eth_address).reverse()
-        }
+        self.eth_address
+            .to_string()
+            .to_lowercase()
+            .cmp(&other.eth_address.to_string().to_lowercase())
```

Comment:
you should be able to sort on bytes instead of going through strings

Reply:
Good catch. Updated on the RUST side, because it's a ([u8; 20]); 
But on Go side it's just a string, so case ignore is important to have the same sorting on RUST and Go sides.